client/lib/create-selector/index.js

```diff
@@ -75,6 +79,10 @@ export default function createSelector( selector, getDependants = DEFAULT_GET_DE
 	let lastDependants;
 
 	return Object.assign( function( state, ...args ) {
+		if ( Array.isArray( getDependants ) ) {
+			getDependants = makeSelectorFromArray( getDependants );
+		}
+
```

Comment:
@aduth this was an experiment, but I kind of like it. Do you think it's a worthwhile addition to `createSelector`?